src/math.rs

```diff
@@ -150,22 +199,6 @@ impl<'a, S: BaseFloat> Mul<&'a Vector3<S>> for &'a Isometry3<S> {
     }
 }
 
-impl<S: BaseFloat> Into<Decomposed<Vector3<S>, Quaternion<S>>> for Isometry3<S> {
```

Comment:
Any reason why the generics were removed here and instead having an `f64` implementation only?

Reply:
this there is indeed some automatic or manual merging gone wrong! I am restoring them

Reply:
It would be clearer to move them back exactly where they were before, so they don't show up as diff...